package.json

```diff
@@ -22,30 +22,33 @@
     }
   ],
   "main": "lib/image-diff",
+  "files": [
+    "lib/"
+  ],
   "engines": {
-    "node": ">= 0.8.0"
+    "node": ">= 0.10.0"
```

Comment:
If we are adjusting the `node` engine, then we should probably adjust it in the same commit as Travis CI adjustments.